packages/ppa-angular/projects/ppa-angular-shared/src/lib/ppa-angular-shared.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ppa-shared-shared',
```

Comment:
duplicate

Reply:
I know it's duplicate but this is the default component created by the CLI and the prefix for the app components is ppa-shared that's why it ended up like that